i18n/ja/docusaurus-plugin-content-docs/current/ref-info/ref-token-refresh.md

```diff
@@ -0,0 +1,54 @@
+- The token can be refreshed many times as long as it's always refreshed before expiration.
+- Publishers send UID2 tokens in the bidstream.
+- When you refresh a UID2 token, this does not invalidate/expire the original or previous UID2 token. You can still use the earlier token until it expires.
+- When the UID2 Operator service receives the refresh token with a request for a new UID2 token, it checks for user opt-out. If the user has opted out of UID2, no new UID2 token is generated. For details, see [User Opt-Out](../getting-started/gs-opt-out.md).
+
+## Refresh Tokens: Key Information
+
+Here are some key points about refresh tokens:
+- A refresh token is an opaque string that is issued along with the <a href="glossary-uid#gl-uid2-token">UID2 token</a>.
+- Refresh tokens are case sensitive.
+- The token value is opaque: do not make any assumptions about the format or about the length of the string.
+- You can use the refresh token to generate a new UID2 token before the old one expires.
+- Token refresh can be managed in a variety of ways, such as:
+  - With a UID2 SDK (see [SDK Functionality](../sdks/summary-sdks.md#sdk-functionality))
+  - By calling the [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md) endpoint
+  - By using the UID2 Prebid.js module (see [UID2 Integration Overview for Prebid.js](../guides/integration-prebid.md))
+- The [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md) endpoint does not require authentication with your UID2 credentials.
+- When a new UID2 token is generated and returned, a new refresh token is returned along with it.
+- As long as you refresh the token before it expires, you can refresh many times. There is no limit as long as the UID2 token and corresponding refresh token are valid.
+- If you refresh the token, and get a new token before the old one has expired, you can use either the new token or the old one, as long as you use a token that is still valid/has not expired.
+- In most cases, you can refresh tokens on the client side, even if the token was generated on the server side. For details about refresh functionality for the various SDKs, see [SDK Functionality](../sdks/summary-sdks.md#sdk-functionality) (Refresh UID2 Token column).
+
+## Recommended Token Refresh Frequency
+
+To determine when to refresh, you can use the timestamp of the `refresh_from` field in the response to the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint (see [Successful Response](../endpoints/post-token-generate.md#successful-response)) or [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md) endpoint (see [Successful Response With Tokens](../endpoints/post-token-refresh.md#successful-response-with-tokens)). The value of this field is a timestamp in <a href="glossary-uid#gl-utc">UTC</a> format.
+
+We recommend refreshing once at startup, and then periodically (recommended refresh interval is hourly).
```

Comment:
do we really recommend refresh hourly? (i can't remember but if you could show me a reference it would be great)

Reply:
No... you're correct and thx a million. My mistake. "recommended refresh interval is hourly" is from sharing keys, not token refresh. But what is it? Hope to get that answer and I'll fix this.